.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
   <state>
-    <option name="PREFERRED_PROJECT_CODE_STYLE" value="Kroxylicious" />
+    <option name="USE_PER_PROJECT_SETTINGS" value="true" />
```

Comment:
IDK what this does, but was it intended?

Reply:
I was seeing a fight between the IntelliJ code formatter and the impsort plugin. This change tells IntelliJ to pickup the format yet settings we already have checked into the project